src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -67,7 +67,7 @@
 
 import static gregtech.api.capability.GregtechDataCodes.*;
 
-public abstract class MetaTileEntity implements ICoverable {
+public abstract class MetaTileEntity implements ICoverable, IVoidable {
```

Comment:
(Not a change to be done here)
Implementing IVoidable here means you can remove it from the single block generators and the large turbine.